releases.yaml

```diff
@@ -20,6 +20,10 @@ lts:
   release_date: "April 2024"
   eol: "April 2029"
   release_notes_url: "https://discourse.ubuntu.com/t/noble-numbat-release-notes/24668"
+  iso_download_size: "6GB"
+  server_iso_size: "2.7GB"
+  raspi_desktop_iso_size: "2.6GB"
+  raspi_server_iso_size: "1GB"
```

Comment:
Not sure how to check these

Reply:
There is a file in the description from which I got all these numbers. There they are shown in bytes, and I rounded them to Gbytes.

Reply:
or is your question how to check if the image that you download correspond to that size?